src/components/views/dialogs/UntrustedDeviceDialog.tsx

```diff
@@ -17,9 +17,20 @@ import BaseDialog from "./BaseDialog";
 import { IDevice } from "../right_panel/UserInfo";
 
 interface IProps {
+    /**
+     * The user whose device is untrusted.
+     */
     user: User;
+    /**
+     * The device that is untrusted.
+     */
     device: IDevice;
-    onFinished(mode?: "legacy" | "sas" | false): void;
+    /**
+     * Callback for when the dialog is dismissed.
+     * If mode is "sas", the user wants to verify the device with SAS. Otherwise, the dialog was dismissed.
```

Comment:
If it's the "callback for when the dialog is dismissed", then the dialog was dismissed whatever `mode` is set to.

```suggestion
     * If mode is "sas", the user wants to verify the device with SAS. Otherwise, the dialog was dismissed normally.
```